files/KoreBuild/modules/vstest/module.targets

```diff
@@ -54,4 +55,13 @@ Runs the VSTest on all projects in the ProjectsToTest itemgroup.
       ContinueOnError="$(_TestContinueOnError)"
       RemoveProperties="$(_BuildPropertiesToRemove)" />
   </Target>
+
+  <Target Name="ValidateBenchmarks" DependsOnTargets="ResolveSolutions" Condition=" '$(EnableBenchmarkValidation)' == 'true' ">
+
+    <Message Text="%0ARunning benchmark validation for:%0A@(BenchmarksToValidate -> '%(FileName)','%0A')%0A"
+      Importance="High"
+      Condition="'@(BenchmarksToValidate)' != ''" />
+
+    <Exec Command="dotnet run --no-build --project %(BenchmarksToValidate.Identity) -- * --validate-fast" IgnoreStandardErrorWarningFormat="true" />
```

Comment:
By the way, dotnet-run has historically been problematic from build targets. A more reliable way to invoke an executable is like this:
```xml
<MSBuild Projects="Proj.csproj" Targets="GetTargetPath">
  <Output TaskParameter="TargetOutputs" PropertyName="ExecPath" />
</MSBuild>
<Exec Command="dotnet $(ExecPath)" />
```